addons/binding/org.openhab.binding.unifi/ESH-INF/thing/thing-types.xml

```diff
@@ -146,4 +147,11 @@
 		<state readOnly="true"></state>
 	</channel-type>
 
+	<channel-type id="blocked">
+		<item-type>Switch</item-type>
+		<label>Blocked</label>
+		<description>Is device blocked</description>
+		<state readOnly="false"></state>
```

Comment:
State readOnly is default `false`. Therefor remove it here.

Reply:
Removed.